eventmesh-spi/src/main/java/org/apache/eventmesh/spi/loader/JarExtensionClassLoader.java

```diff
@@ -50,33 +50,34 @@ public class JarExtensionClassLoader implements ExtensionClassLoader {
 
     private static final String EVENT_MESH_PLUGIN_DIR = "eventMeshPluginDir";
 
-    private static final ConcurrentHashMap<Class<?>, Map<String, Class<?>>> EXTENSION_CLASS_CACHE =
-            new ConcurrentHashMap<>(16);
+    private static final String EVENTMESH_EXTENSION_PLUGIN_DIR = System.getProperty(EVENT_MESH_PLUGIN_DIR,
+        Joiner.on(File.separator).join(Lists.newArrayList(".", "plugin")));
 
-    private static final String EVENTMESH_EXTENSION_PLUGIN_DIR =
-            System.getProperty(EVENT_MESH_PLUGIN_DIR,
-                    Joiner.on(File.separator).join(Lists.newArrayList(".", "plugin")));
+    private static final JarExtensionClassLoader INSTANCE = new JarExtensionClassLoader();
+
+    private final ConcurrentHashMap<Class<?>, Map<String, Class<?>>> extensionClassCache = new ConcurrentHashMap<>(16);
```

Comment:
Why not use static modifier?

Reply:
@jonyangx JarExtensionClassLoader is a singleton mode, and extensionClassCache is a property